mrmustard/physics/fock.py

```diff
@@ -836,34 +887,17 @@ def quadrature_distribution(
     Returns:
         tuple(Vector, Vector): coordinates at which the pdf is evaluated and the probability distribution
     """
-    dims = len(state.shape)
-    if dims > 2:
-        raise ValueError(
-            f"Input state has dimension {state.shape}. Make sure is either a single-mode ket or dm."
-        )
-
-    is_dm = dims == 2
     cutoff = state.shape[0]
-
-    if not np.isclose(quadrature_angle, 0.0):
-        # rotate mode to the homodyne basis
-        theta = -math.arange(cutoff) * quadrature_angle
-        Ur = math.diag(math.make_complex(math.cos(theta), math.sin(theta)))
-        state = (
-            math.einsum("ij,jk,kl->il", Ur, state, math.dagger(Ur))
-            if is_dm
-            else math.matvec(Ur, state)
-        )
-
     if x is None:
         x = np.sqrt(settings.HBAR) * math.new_constant(estimate_quadrature_axis(cutoff), "q_tensor")
 
-    psi_x = math.cast(oscillator_eigenstate(x, cutoff), "complex128")
-    pdf = (
-        math.einsum("nm,nj,mj->j", state, psi_x, psi_x)
-        if is_dm
-        else math.abs(math.einsum("n,nj->j", state, psi_x)) ** 2
-    )
+    dims = len(state.shape)
+    is_dm = dims == 2
+
+    quad = math.transpose(math.astensor([x] * dims))
+    conjugates = [False, True] if is_dm else [False]
```

Comment:
I already commented above, but the convention is that bras come before kets so this may be the opposite of what we want

Reply:
Good catch!